lib/bulk_insert/worker.rb

```diff
@@ -16,12 +16,14 @@ def initialize(connection, table_name, primary_key, column_names, set_size=500,
       @ignore = ignore
       @update_duplicates = update_duplicates
       @return_primary_keys = return_primary_keys
+      @ignored_columns_on_update = Array.wrap(ignored_columns_on_update)
```

Comment:
@jamis I am not sure if the gem is supposed to work only with a dependency on `activerecord` or it was designed to have a dependency on `activesupport` as well:

https://apidock.com/rails/Array/wrap/class

Reply:
Ah, good spot. I don't think we need to keep this, and can use `Array(ignored_columns..)` as we are not expecting anything unusual in that param like a `hash`.